server/src/controllers/maproom/v2/cells/homeCell.ts

```diff
@@ -13,11 +14,12 @@ export const homeCell = async (ctx: Context) => {
    * visit the Wiki page: 'World Map v2: Cell Object'
    */
   return {
-    uid: save.userid, // ToDo: Why do we have a userId for both user and save table? Fix
+    cell_id: cell.cell_id,
+    uid: cell.uid, // ToDo: Why do we have a userId for both user and save table? Fix
     b: 2,
     fbid: save.fbid,
     pi: 0,
-    bid: save.homebaseid,
+    bid: save.basesaveid,
```

Comment:
Hmm... are you sure about this change? I will need some explanation. If you look in `MapRoomCell.as` we have this:

```javascript
         if(serverData.bid)
         {
            if(this._baseID != 0 && this._baseID == GLOBAL._homeBaseID)
            {
               MapRoom._homeCell = this;
            }
            else if(this.X == GLOBAL._mapHome.x && this.Y == GLOBAL._mapHome.y)
            {
               MapRoom._homeCell = this;
            }
            this._baseID = serverData.bid;
         }
```

It seems it checks if `bid` is equal to `homebaseid`, is there any reason why we have set it to be `basesaveid`?

Reply:
I reverted it back to homebaseid